README.md

```diff
@@ -37,7 +37,8 @@ Python.
 
 ```bash
 git clone https://github.com/juliema/aTRAM.git
-pip install --user --requirement atram/requirements.txt
+cd aTRAM
+python3 setup.py install
```

Comment:
I prefer to avoid global installs and do things like you did for the conda install. That is, we can either use a `--user` switch on the the setup.py or, even better still use the setup.py with a virtual environment.

Reply:
I agree and now the entire installation can be obtained by using a `conda` environment